admin/spec/service/data-access/pupil.data.service.spec.js

```diff
@@ -1,5 +1,5 @@
 'use strict'
-/* global describe, beforeEach, afterEach, it, expect */
+/* global describe, beforeEach, afterEach, it, expect spyOn */
 
 const proxyquire = require('proxyquire').noCallThru()
```

Comment:
is this actually used anymore here?

Reply:
Yes, in many places.

Reply:
i was referring to proxyquire. probably didnt see it in the reduced diff

Reply:
that's what I was referring to.  It' still used in the spec.